client/state/followers/thunks/fetch-followers.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+
+import debugFactory from 'debug';
+
+const debug = debugFactory( 'calypso:followers-actions' );
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'calypso/lib/wp';
+import { requestFollowers, successRequestFollowers, failRequestFollowers } from '../actions';
+
+/**
+ * Request followers for a given site.
+ *
+ * @param {object} query contains params to filter results
+ * @param {boolean} silentUpdate indicates whether the request should be reflected in the UI
+ */
+function fetchFollowers( query, silentUpdate = false ) {
+	return ( dispatch ) => {
+		// TODO: Componentes should not fetch if already fetching
```

Comment:
Ditto. Data layer is a good way to fix this. 😉

Reply:
done via d056358